src/main/webapp/tests.html

```diff
@@ -15,7 +15,6 @@
     <div id="menu"></menu>
   </div>
   <script src="https://code.jquery.com/qunit/qunit-2.10.0.js"></script>
-  <script src="https://unpkg.com/qunit-dom/dist/qunit-dom.js"></script>
```

Comment:
Keep this line?

Reply:
This line shows up twice, I was deleting one of the copies. Should have left a comment about it, my bad!